include/zephyr/dt-bindings/pinctrl/nrf-pinctrl.h

```diff
@@ -126,6 +126,28 @@
 #define NRF_FUN_QSPI_IO2 33U
 /** QSPI IO3 */
 #define NRF_FUN_QSPI_IO3 34U
+/** EXMIF CK */
+#define NRF_FUN_EXMIF_CK 35U
+/** EXMIF DQ0 */
+#define NRF_FUN_EXMIF_DQ0 36U
+/** EXMIF DQ1 */
+#define NRF_FUN_EXMIF_DQ1 37U
+/** EXMIF DQ2 */
+#define NRF_FUN_EXMIF_DQ2 38U
+/** EXMIF DQ3 */
+#define NRF_FUN_EXMIF_DQ3 39U
+/** EXMIF DQ4 */
+#define NRF_FUN_EXMIF_DQ4 40U
+/** EXMIF DQ5 */
+#define NRF_FUN_EXMIF_DQ5 41U
+/** EXMIF DQ6 */
+#define NRF_FUN_EXMIF_DQ6 42U
+/** EXMIF DQ7 */
+#define NRF_FUN_EXMIF_DQ7 43U
+/** EXMIF CS0 */
+#define NRF_FUN_EXMIF_CS0 44U
+/** EXMIF CS1 */
+#define NRF_FUN_EXMIF_CS1 45U
```

Comment:
content in the wrong commit?

Reply:
This is the first commit that makes use of those definitions, so I put them here. Should those definitions be in a separate commit?

Reply:
How does the pinctrl driver configure exmif?

Reply:
On nRF54H20 it's not needed. The configuration is done by nrf-regtool.

Reply:
I think it would be better to add this definitions in a separate commit, with a note in its message explaining why no changes in `pinctrl_nrf.c` are needed.

Reply:
> I think it would be better to add this definitions in a separate commit, with a note in its message explaining why no changes in `pinctrl_nrf.c` are needed.

agreed

Reply:
Split the change to a separate commit.